pulp_rpm/app/models.py

```diff
@@ -895,6 +942,26 @@ def grplist_to_lst(cls, value):
                              'default': i.default})
         return grp_list
 
+    @classmethod
+    def list_to_idlist(cls, list):
```

Comment:
Nit picking. It's usually preferable (and in some cases necessary) not to use variable names which are the same as for some python standard objects. 
How about `lst`? or something more specific like `group_ids`?

Reply:
Good call, made the change.